src/components/NewDraftWizard/WizardComponents/WizardObjectIndex.js

```diff
@@ -283,11 +310,17 @@ const WizardObjectIndex = (): React$Element<any> => {
               id="type-header"
             >
               <NoteAddIcon /> <Typography variant="subtitle1">{typeCapitalized}</Typography>
-              <Badge
-                badgeContent={getDraftCount("draft-" + objectType)}
-                className={classes.badge}
-                data-testid="badge"
-              />
+              {getSavedObjectCount(objectType) > 0 && (
+                <Tooltip title="Saved objects">
```

Comment:
Can we change the `tooltip` title to "Submitted objects" or something else ?
`Saved objects` sounds a bit confusing I think, because we submitted the objects already (?)

Reply:
Sounds good, I'll make the change.